heltour/tournament/tasks.py

```diff
@@ -22,89 +49,95 @@
 import reversion
 import textwrap
 import time
-import sys, traceback
+import sys
+import traceback
 
 logger = get_task_logger(__name__)
 
 UsernamesQuerySet = ValuesQuerySet[Player, Dict[str, str]]
 
+
 def to_usernames(users: UsernamesQuerySet) -> List[str]:
```

Comment:
I think that using https://docs.djangoproject.com/en/5.2/ref/models/querysets/#values-list with `flat=True` is probably faster and more Djangonic (pythonic??) than this list.